src/main/kotlin/uk/gov/justice/digital/hmpps/oneloginuserregistry/model/entity/BookerPrisoner.kt

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.digital.hmpps.oneloginuserregistry.model.entity
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.OneToMany
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.CreationTimestamp
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "booker_prisoner")
+class BookerPrisoner(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "ID")
+  val id: Long = 0,
+
+  @Column(name = "booker_id", nullable = false)
+  val bookerId: Long,
+
+  @ManyToOne
+  @JoinColumn(name = "booker_id", updatable = false, insertable = false)
+  val booker: Booker,
```

Comment:
Were is the other part of this relationship in the Booker
`val prisoners: MutableList<BookerPrisoner> = mutableListOf(),`

Reply:
thanks, added to BOOKER.